src/components/frame/Extensions/pages/ExtensionDetailsPage/PermissionsNeededBanner.tsx

```diff
@@ -7,31 +7,50 @@ const displayName = 'frame.Extensions.PermissionsNeededBanner';
 const PermissionsNeededBanner = () => {
   // @TODO: Change extension missing permissions functionality
   const [isPermissionEnabled, setIsPermissionEnabled] = useState(false);
-  return (
-    <div className="mb-6">
+
+  function getBanner() {
```

Comment:
Please change to arrow function for consistency (unless there's a reason for `function`)